packages/tables/resources/views/index.blade.php

```diff
@@ -184,6 +185,13 @@ class="mr-2"
                                 class="shrink-0"
                             />
                         @endif
+
+                        @if ($isColumnToggleFormVisible)
+                            <x-tables::toggleable
+                                :form="$getColumnToggleForm()"
```

Comment:
Think we need to pass in a width here based on the number of columns the form has...

Reply:
Maybe it will be ok for just 1 column in the columns list for now, until it we get other requests. People try to minimise the number of columns in tables anyway to avoid scroll.

Reply:
```suggestion
                                :form="$getColumnToggleForm()"
                                :width="$getColumnToggleFormWidth()"
```